content/show/coder-radio/0000.md

```diff
@@ -0,0 +1,43 @@
+{
+  "type": "episode",
+  "draft": false,
+  "show_slug": "coder-radio",
+  "show_name": "Coder Radio",
+  "episode": 0,
+  "episode_padded": "0000",
+  "episode_guid": "00000000-0000-0000-0000-000000000000",
+  "slug": "0",
+  "title": "New Show!",
+  "description": "A weekly talk show taking a pragmatic look at the art and business of software development and related technologies. Premieres LIVE June 11th 2012 at 9am PDT / Noon EDT. Available for download early June 12th!",
+  "date": "2012-06-09",
+  "header_image": "/images/shows/coder-radio.png",
+  "categories": [
+    "Coder Radio"
+  ],
+  "tags": [],
+  "hosts": [
+    "chris",
+    "michael"
+  ],
+  "guests": [],
+  "sponsors": [],
+  "podcast_duration": null,
+  "podcast_file": null,
+  "podcast_bytes": null,
+  "podcast_chapters": null,
+  "podcast_alt_file": "https://traffic.libsyn.com/jnite/thedust-mp3.mp3",
+  "podcast_ogg_file": "https://traffic.libsyn.com/jnite/thedust-ogg.ogg",
```

Comment:
This link doesn't point to a file anymore, including anything with `libsyn.com/jnite` (Jupiter@Nite, the old JB show?)